<?xml version="1.0" encoding="UTF-8"?>
<record_update table="sys_scope_privilege">
    <sys_scope_privilege action="INSERT_OR_UPDATE">
        <operation>execute</operation>
        <source_scope display_value="Citrix IT Service Management Connector">790cd3e8db8513406d3c72fc0f961915</source_scope>
        <status>allowed</status>
        <sys_class_name>sys_scope_privilege</sys_class_name>
        <sys_created_by>sebastiang</sys_created_by>
        <sys_created_on>2018-03-27 00:19:18</sys_created_on>
        <sys_customer_update>false</sys_customer_update>
        <sys_id>1accb9d2db0997406d3c72fc0f961985</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>ScriptableRESTResponse.haveError</sys_name>
        <sys_package display_value="Citrix IT Service Management Connector" source="x_wcs_ctrx_auto_sv">790cd3e8db8513406d3c72fc0f961915</sys_package>
        <sys_policy/>
        <sys_replace_on_upgrade>false</sys_replace_on_upgrade>
        <sys_scope display_value="Citrix IT Service Management Connector">790cd3e8db8513406d3c72fc0f961915</sys_scope>
        <sys_update_name>sys_scope_privilege_1accb9d2db0997406d3c72fc0f961985</sys_update_name>
        <sys_updated_by>sebastiang</sys_updated_by>
        <sys_updated_on>2018-03-27 00:19:18</sys_updated_on>
        <target_name>ScriptableRESTResponse.haveError</target_name>
        <target_scope display_value="Global">global</target_scope>
        <target_type>scriptable</target_type>
    </sys_scope_privilege>
</record_update>
